scss/mixins/_grid.scss

```diff
@@ -92,6 +87,11 @@
     $infix: breakpoint-infix($breakpoint, $breakpoints);
 
     @include media-breakpoint-up($breakpoint, $breakpoints) {
+      // Provide basic `.col-{bp}` classes for equal-width flexbox columns
+      .col#{$infix} {
```

Comment:
IMHO we should skip if `$infix` is empty, and prefer to set `.col` before any loop. Maybe hard-coded in our main grid file, maybe?